cli/ops/dispatch_json.rs

```diff
@@ -103,15 +103,7 @@ where
   if is_sync {
     Ok(JsonOp::Sync(f()?))
   } else {
-    // TODO(ry) use thread pool.
-    let fut = crate::tokio_util::spawn_thread(f);
-    /*
-      let fut = async move {
-        tokio::task::spawn_blocking(move || f())
-          .await
-          .map_err(ErrBox::from)?
-      }.boxed_local();
-    */
+    let fut = async move { tokio::task::spawn_blocking(f).await.unwrap() };
```

Comment:
Is this actually spawning in some threadpool? I thought `spawn_blocking` would be spawning on the current runtime, which (if I am understanding correct, I am trying to catch up with recent changes) would be the single-threaded one we created through `run_basic` at the very beginning? Did I miss something here?

Reply:
To examine this I used dtrace to print each of the syscalls and which thread they fall on. The program I'm running is `await Deno.mkdir("/tmp/foo");`

Here's the output:

```
> sudo dtrace -n 'syscall:::entry { printf("%d %s\n", tid, probefunc); }' -c './target/debug/deno -A x.js' -q
dtrace: system integrity protection is on, some features will not be available

2342926 ioctl
2342926 ioctl
2342926 sysctl
2342926 sysctl
....
```
Full output is here: https://gist.github.com/ry/aa5ae51d1cac8212ef50d3b8cb026564

If you look for mkdir and then search for the thread id associated with it, you see it only appears in a little block and is not the main thread id.
<img width="271" alt="Screen Shot 2020-02-07 at 5 48 58 PM" src="https://user-images.githubusercontent.com/80/74071553-3e92f680-49d2-11ea-85af-1ed33bc5b318.png">


So it appears to me to actually be using the thread pool.

Reply:
Yep I realized that. Sorry for the confusion